sbin/nvmecontrol/nvmecontrol.8

```diff
@@ -998,13 +998,13 @@ If the
 .Dq nvmecontrol identify -n 1 nvme2
 command ended with
 .Pp
-.Bl -verbatim
+.Bd -literal
```

Comment:
There's no `.Bl -verbatim`; it seems clear to me that the original author intended `.Bd -literal` here.

(This results in different `<div>` classes and a `<pre>` in the html output, but that's a desired change.  No affect in the ascii output.)